### ISSUE TITLE
Police Oversight: Fix spanish translations

### ISSUE THREAD

**Issue by sergiogcx** [cityofaustin/techstack#1238]
Somehow the translations are causing a problem (translating code), so far the only i was able to determine causing a problem is "Back" and "Continue". So it is matter of adding a few `%` symbols here and there.

![Screen Shot 2019-02-19 at 6.42.26 PM.png](https://images.zenhubusercontent.com/5b7edad7290aac725aec290c/192a385b-ca3e-48da-858e-5a1f24124054)

**Comment by sergiogcx** [comment]
I just tested it locally and it works now, time to merge to master.

**Comment by sergiogcx** [comment]
After speaking with @desigonz we are going to incorporate the changes into her PR and use that for deployments:

https://github.com/cityofaustin/officer-complaint-form/pull/40

Once this file is merged into master, we will be able to fix the deployment problem.

**Comment by desigonz** [comment]
@sergiogcx just finished incorporating your %'s

**Comment by sergiogcx** [comment]
wow that was blazing fast!

**Comment by sergiogcx** [comment]
Changing status to Review Q/A, the form works and it is being deployed correctly.

**Comment by desigonz** [comment]
Awesome, thanks for merging! One thing I'm noticing (which is a content team thing and NOT a high priority): our style is to use sentence case for headings, so "File a complaint" and "Presente una queja" (not "File a Complaint" and "Presente Una Queja"). I think some of those just got changed—mind if I go back and fix those?
